docs/process.md

```diff
@@ -173,16 +173,10 @@ Their full descriptions can be viewed under the [labels page](https://github.com
 #### Code review
 
 Your code will be reviewed, in this sequence, by:
-* Travis CI: by running static analysis.<br>
```

Comment:
This part is still accurate and should not be removed. The only inaccurate part is the line after this ("If there are problems found, the build will terminate without proceeding to testing") and obviously Travis CI -> GitHub Actions.